### PR TITLE
Adding Support for Sync CLI

### PR DESCRIPTION
**Pull request by Amit3200**
- Adding Support for Sync CLI
- Returns the comparison details if the builds are running

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "webdriverio": "~6 || ~7 || ^8.6.6"
```
Comment (chinmay-browserstack) [inline package.json:32]: https://github.com/percy/percy-webdriverio/pull/1113/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L29 also need to be updated

File: index.js [hunk 2]
```diff
@@ -49,6 +50,11 @@ module.exports = function percySnapshot(b, name, options) {
   });
 };
 
+// To mock the test case
+module.exports.request = async function request(data) {
```
Comment (this-is-shivamsingh) [inline index.js:54]: we can spyOn(utils, postSnapshot) as well

**Comment by this-is-shivamsingh** [comment]
I can see we don't have support for percyScreenshot with wdio ?

File: index.js (hunk 2 above)
Comment (Amit3200) [reply index.js:54]: This will not work as utils does not have setter and getter it will need the cli update and code change for the same.

**Comment by Amit3200** [comment]
> I can see we don't have support for percyScreenshot with wdio ?

percyScreenshot is supported via `percy-selenium-js` for wdio.

**Review by shantanuk-browserstack — APPROVED** [review]

**Review by prklm10 — APPROVED** [review]

**Pull request merged**